protocol/avdl/keybase1/notify_fs.avdl

```diff
@@ -6,5 +6,9 @@ protocol NotifyFS {
   @notify("")
   void FSActivity(FSNotification notification);
 
+  void FSSyncActivity(FSPathSyncStatus status) oneway;
```

Comment:
I agree. but I'm just keeping it consistent with how the file was originally done.  A change here would propagate to too many places for this PR.  Agreed we should clean it up in the future.